### DISCUSSION TITLE
General Design Discussion

### DISCUSSION THREAD

**Discussion by GarthDB** [DesignSystemsEngineer/designsystemsengineer.github.io#38] [General]
1. How do we go about designing the website?
2. Should we start with a theme as a placeholder?
3. How do we give designers a chance to work on it?
4. Do we build a design system for the site?
5. What do we name the design system? (The most important question)
6. Should we make merch? (pretty important question)

**Comment by ddamato** [comment]
I imagine there will be people who don't like these answers but:

1. Start with a template
2. See previous answer
3. You don't (yet)
4. Too early so no
5. Cross the bridge later
6. Probably maybe sure?

In my opinion, content is king. The look and feel of the site can enhance what the content is. Only after there is content does that inform the presentation of said content.

**Reply by nklansford** [reply]
I agree with this. Customize an existing template for the brand. Make merch (maybe designers can help with merch if you want help there... everyone likes a good sticker... I like a good sticker). 

If there's a design system at this point, I would recommend it be for an example project for content/educational purposes–not to maintain the site. Then you can change things as often as you need to explain concepts you'd like to cover without it impacting your site. If you went that route, you could have designers collaborate on it whenever you want to engage with that side of DS work.

**Reply by GarthDB** [reply]
@nklansford agreed. You beat me to some of my points.
@ddamato thank you for the comments. I really agree with the general sentiment and idea of staying focused on content, especially to start. Because we spend so much time in the profit-driven business mindset, I keep having to remind myself that this project isn't going to work exactly like a business. I think our most valuable commodity is the engagement and interest of individuals. This could be consumers of content, authors, designers, or any type of contributor. If I could get some design system designers to feel inspired to start considering and contributing design work and discussion at any stage of this project, that would be a win. I'm sure this will result in rework in the future, and I will embrace that as well because it's further engagement.

Also, and this is a bit more philosophical, I believe every project/product has a design system; some are better defined and maintained than others. Starting with a template would be the first seed of the design system. Normally, I would do this sort of thing in a small group and be less open about the process. With Design Systems Engineer, I'm trying to see what it would be like to be open and publicly collaborative from the beginning.

This is a round about way of saying, thank you for your input, I agree, let's start with a template, but also, let's at least think about what comes after that.

**Reply by somerandomdude** [reply]
Yeah, I think the biggest/best thing you can do is get quality content up. As long as it doesn't make readers' eyes bleed, it's mission accomplished.

**Comment by mikemai2awesome** [comment]
Make it accessible.

**Reply by GarthDB** [reply]
Thanks @mikemai2awesome. At this stage, I hope that WCAG 2.2 would be the baseline, table stakes, etc., but it's good not to leave that as just an assumption because it's so important. We will need to include some language around accessibility, specifically in the contributing guidelines moving forward.

**Comment by jnjosh** [comment]
> How do we go about designing the website?

Just want to push to include mobile/native content on this. Happy to help there!

**Reply by GarthDB** [reply]
Yes! An oft overlooked aspect of design systems discussions is how to share design data with non-web platforms/implementations.

**Reply by somerandomdude** [reply]
Please! I'd argue that would be some of the most valuable content!

**Reply by GarthDB** [reply]
@somerandomdude, @jnjosh is one of the more experienced in the area. He's been a part of a wonderful effort to have the iOS implementation of [Spectrum](https://spectrum.adobe.com) use the same design tokens and data as the web and design implementations. I'm hoping he would be able to share his insight and experience.
